### PR TITLE
Add meeting ping list and script

### PR DESCRIPTION
**Pull request by gotmax23**
Here is the meeting ping list, as promised :). I can keep the script locally if we don't want it here.

### CODE REVIEW

File: meeting/meeting-people.txt [hunk 1]
```diff
@@ -0,0 +1,27 @@
+acozine
+andersson007_
```
Comment (felixfontein) [inline meeting/meeting-people.txt:2]:
```suggestion
andersson007_
anwesha
```

File: meeting/meeting-people.txt [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+geerlingguy
+gundalow
+gwmngilfen
+ikhan_
+jillr
+jtanner
+lmodemal
+maxamillion
+misc
+nitzmahone
```
Comment (felixfontein) [inline meeting/meeting-people.txt:22]:
```suggestion
nitzmahone
oranod
```

File: meeting/meeting-people.txt [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+dmsimard
+felixfontein
+gotmax
+geerlingguy
+gundalow
+gwmngilfen
+ikhan_
+jillr
+jtanner
+lmodemal
```
Comment (felixfontein) [inline meeting/meeting-people.txt:19]:
```suggestion
lmodemal
mariolenz[m]
```

File: meeting/meeting-people.txt [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+maxamillion
+misc
+nitzmahone
+resmo
+samccann
+tadeboro
```
Comment (felixfontein) [inline meeting/meeting-people.txt:25]:
```suggestion
```

File: meeting/README.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+#ansible-community (libera.chat) /
+[#community:ansible.com](https://matrix.to/#/#community:ansible.com) (Matrix).
+
+## Ping List
+
+Interested community members can add their IRC/Matrix nick to
+[meeting-people.txt](https://github.com/ansible-community/community-topics/blob/main/meeting/meeting-people.txt)
+if they'd like to be pinged when the meeting starts.
+
+The meeting chair can run ./meeting-people.py to format the list into a ping
```
Comment (felixfontein) [inline meeting/README.md:13]:
```suggestion
The meeting chair can run `./meeting-people.py` to format the list into a ping
```

**Comment by felixfontein** [comment]
I like the idea of having the script here as well.

File: meeting/meeting-people.txt (hunk 4 above)
Comment (gotmax23) [reply meeting/meeting-people.txt:25]: Hmm, I thought I removed tadeboro already, but I guess not.

File: meeting/meeting-people.txt [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+ikhan_
+jillr
+jtanner
+lmodemal
+mariolenz[m]
+maxamillion
+misc
+nitzmahone
+oranod
+resmo
```
Comment (russoz) [inline meeting/meeting-people.txt:26]:
```suggestion
resmo
russoz
```

**Review by russoz — COMMENTED** [review]
Please add me as well. The meeting times are frequently not the best for my TZ, but if I get the ping and am available, I will hop in.

File: meeting/meeting-people.txt [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+felixfontein
+gotmax
+geerlingguy
+gundalow
+gwmngilfen
+ikhan_
+jillr
+jtanner
+lmodemal
+mariolenz[m]
```
Comment (markuman) [inline meeting/meeting-people.txt:21]:
```suggestion
mariolenz[m]
markuman
```

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@gotmax23 thanks for doing this!

**Pull request merged**